types/react-tabs-redux/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
Remove this rule from here and fix the error. New packages cannot special case tslint file.